src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.NotEnoughAgeException;
+import core.basesyntax.exceptions.NotEnoughSizeException;
+import core.basesyntax.exceptions.UserIsNullException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static final User EXISTING_USER = new User("BobAlison", "password", 25);
+    private static final User NEW_USER_WITH_EXISTING_LOGIN = new User(
+            "BobAlison", "password", 30);
+    private static final User NULL_USER = null;
+    private static final User VALID_USER = new User("NickAlison", "password", 65);
+    private static final User USER_WITH_NULL_AGE = new User("BobAlison", "password", null);
+    private static final User USER_WITH_NULL_PASSWORD = new User("BobAlison", null, 25);
```

Comment:
OPTIONAL: In my opinoin there is no need to hardcode users. It is better to create constants like validPassword, invalidPassword, validAge, invalidAge etc and just create users from constant blocks like lego